example/index.ts

```diff
@@ -57,11 +57,7 @@ document.addEventListener('DOMContentLoaded', async () => {
 
     // update displayed doc
     exampleContainer.innerHTML = md.render(
-      page.html.replace(
-        /\${package_version}/g,
-        // @ts-expect-error global variables
-        process.env.npm_package_version,
```

Comment:
It looks like there is no `.env` file, I suppose that's why this was coming through as undefined? But it seems like we want to grab the version from `package.json` instead anyway 👍🏻

Reply:
I have an .env.local for the google maps api key and I think (I don't really remember) the prod one gets added automagically with the github actions and uses secrets from the repo. It still wasn't working with the .env.local for me.